server/api/dev.js

```diff
@@ -0,0 +1,214 @@
+/*
+  Easier to refresh entire dev entry instead of upsert/update the dates
+  ??? What is the removed column for in entries?
+*/
+
+const db = require('../db');
+const { createLogger } = require('../logger');
+
+const NOW = new Date();
```

Comment:
Do you know if this stays constant for the entire run of the server since it's defined outside of the function? Or is instantiated again every time a function in the file is called? This sounds like one of those interview questions, and I'm not sure what the answer is.

Reply:
Good catch! It does stay the same. I fixed this.